src/core/thread/lowpan.cpp

```diff
@@ -422,14 +422,42 @@ int Lowpan::CompressUdp(Message &aMessage, uint8_t *aBuf)
 {
     Ip6::UdpHeader udpHeader;
     uint8_t *cur = aBuf;
+    uint8_t *udpCtl = cur;
+    uint16_t source;
+    uint16_t destination;
 
     aMessage.Read(aMessage.GetOffset(), sizeof(udpHeader), &udpHeader);
+    source = udpHeader.GetSourcePort();
+    destination = udpHeader.GetDestinationPort();
 
     cur[0] = kUdpDispatch;
     cur++;
 
```

Comment:
Suppose source is 0xf000, destination is 0xf0b0,  what should the compression look like?